### PR TITLE
Update Project.toml

### PR DESCRIPTION
**Pull request by clad26**
[Bumped compat of Adapt to 3.1]

### PR Checklist

- [ ] Tests are added
- [ ] Entry in NEWS.md
- [ ] Documentation, if applicable
- [ ] Final review from `@dhairyagandhi96` (for API changes).

### CODE REVIEW

**Comment by DhairyaLGandhi** [comment]
I'm not sure if released cuda is expected to work with Adapt@3

**Comment by DhairyaLGandhi** [comment]
Whoop, seems like it is. Thanks!

File: Project.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
 AbstractTrees = "0.3"
-Adapt = "2.0, 3.0"
+Adapt = "2.0, 3.1"
```
Comment (DhairyaLGandhi) [inline Project.toml:30]:
```suggestion
Adapt = "2.0, 3"
```

For the Pkg resolver error.

**Review by DhairyaLGandhi — APPROVED** [review]

**Comment by CarloLucibello** [comment]
"3.0" should be equivalent to "3"

**Pull request closed without merging**